lib/db.php

```diff
@@ -139,6 +145,7 @@ static public function connect($site_config) {
         try {
             self::$dbh[$key] = new PDO($dsn, self::$config['db_user'], self::$config['db_pass']);
             self::$dbh[$key]->setAttribute(PDO::ATTR_EMULATE_PREPARES, false);
+            self::$dbh[$key]->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
```

Comment:
If exceptions are not handled properly, detailed error messages could be displayed to users, exposing sensitive information like SQL queries or database structure. This can lead to security vulnerabilities.

Reply:
That's correct but we still prefer working in exception mode and catch exceptions in relevant places. We have more flexibility this way and errors are not silently ignored.